### PR TITLE
 Added typescript sample app to keploy docs repo

### PR DESCRIPTION
**Pull request by mukul314**
Updated the keploy docs repo by adding typescript sample app in the sample app lists

### CODE REVIEW

**Comment by mukul314** [comment]
@nehagup I have added the sample typescript app, please check and see if its okay.

File: docs/typescript/quickstart/node-fetch.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  - sdk
+  - tutorial
+---
+
+#  Sample Node-fetch App 
+
+A sample node-fetch app containing GET and POST request integrated with keploy's [typescript-sdk](https://github.com/keploy/typescript-sdk)
+
+## Installation
+
```
Comment (SanskritiHarmukh) [inline docs/typescript/quickstart/node-fetch.md:17]: @mukul314 can you include the installation part here instead which you've added [here](https://github.com/keploy/docs/pull/162/files#diff-68f73088dc8486a51efc8eda10a3d5165a5173eaa8569e38c5656d0d17d5cd0cR25) like shown below.
Screenshot from 2023-04-04 20-36-53

Comment (SanskritiHarmukh) [inline docs/typescript/quickstart/node-fetch.md:17]: @mukul314 can you include the installation part here instead which you've added [here](https://github.com/keploy/docs/pull/162/files#diff-68f73088dc8486a51efc8eda10a3d5165a5173eaa8569e38c5656d0d17d5cd0cR25) like shown below.
Screenshot from 2023-04-04 20-36-53

File: docs/typescript/quickstart/node-fetch.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+Now both these API calls were captured as a testcase and should be visible in "keploy-tests" folder in project directory.
+If you're using Keploy cloud, open [this](https://app.keploy.io/testlist).
+
+You should be seeing ` readable/editable yaml files` with their mocks in seperate yamls in "/mock" folder.
+
+
+Now, let's see the magic! 🪄💫
+
```
Comment (SanskritiHarmukh) [inline docs/typescript/quickstart/node-fetch.md:82]: Can you include some images of keploy server when it shows the recorded test cases and mocks when API calls are made?

File: docs/typescript/quickstart/node-fetch.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+test run completed { run id:  bf98b0ed-5793-4280-af60-0b022d533611  }, passed overall:  true
+```
+
+So no need to setup dependencies like mongoDB, web-go locally or write mocks for your testing.
+
+**The application thinks it's talking to the
+server 😄**
+
+Go to the Keploy server logs to get deeper insights on what testcases ran, what failed.
+
```
Comment (SanskritiHarmukh) [inline docs/typescript/quickstart/node-fetch.md:120]: Introducing a bug and failed test cases part is missing, can you figure that out ?

**Review by SanskritiHarmukh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**